recipe/meta.yaml

```
@@ -27,12 +27,13 @@ requirements:
     - {{ pin_compatible('numpy') }}
     - python
     - scipy
-    - pymbar
+    - pymbar <4
     - future
     - networkx
     - lxml
   run_constrained:
     - openforcefield ==9999999999
+    - openff-toolkit <0.11
```

```suggestion
    - openff-toolkit <0.11.0a0
```

This might be marginally safer, but not necessary as the off-main labels should be hidden in any production environments.